app/models/audit_trail.rb

```diff
@@ -15,21 +15,51 @@ def changed_fields
     relevant_fields = modified.map do |key, _value|
       key unless IRRELEVANT_FIELDS.include? key
     end
-
     relevant_fields.compact.map(&:humanize)
   end
 
   # TODO: properly render null values like in special circumstances
   # Something like this should work:
   # "HAHA I WIN" if (fields.all?{|f| f.blank? || (f.flatten.all? &:blank? if f.respond_to?('each'))})
+
+  def formatDates(hash)
```

Comment:
thumbs up! Can you change the method name to use snake case? e.g. `format_dates`